### DISCUSSION TITLE
NordVPN

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#14] [General]
# NordVPN

Check if NordVPN is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/78ac79c9-da8f-4b35-9fb6-1d57ee611882

**Comment by OpenSource03** [comment]
More info: https://www.reddit.com/r/Surface/comments/1dkpoka/nordvpn_response_on_why_their_windows_app_doesnt/

**Comment by talynone** [comment]
Nord has announced they're testing a native client:
https://www.androidauthority.com/copilot-plus-pcs-vpn-3452723/

Both OpenVPN and Wireguard (both have native ARM clients) can be used for now as a work-around: https://nordvpn.com/blog/nordvpn-config-files/

**Comment by talynone** [comment]
Native ARM64 app now available:
https://nordvpn.com/blog/windows-arm-release/

I tested the download, it properly installs the ARM64 version:
https://nordvpn.com/download/windows/

**Reply by OpenSource03** [reply]
Updated, ty
